BUILD.plz

```diff
@@ -1,7 +1,12 @@
+subinclude("//build_defs:version")
+
 go_binary(
     name = "wollemi",
     srcs = ["main.go"],
     visibility = ["PUBLIC"],
+    definitions = {
+        "github.com/tcncloud/wollemi/ports/wollemi.WollemiVersion": VERSION,
```

Comment:
We could also just avoid the genrule and build_defs and just hard code the version here ?